<launch>
  <node name="so_dso_bow" pkg="so_dso_place_recognition" type="test_bow" output="screen">
    <param name="voc_file" value="$(find so_dso_place_recognition)/place_recognition/BoW/ORBvoc.txt" />
    <param name="incoming_id_file1" value="$(find so_dso_place_recognition)/place_recognition/results/KITTI/00/incoming_id_file.txt" />
    <param name="incoming_id_file2" value="$(find so_dso_place_recognition)/place_recognition/results/KITTI/00/incoming_id_file.txt" />
    <param name="bag1" value="/home/moxxx066/Workspace/data/bags/KITTI/seq00.bag" />
    <param name="bag2" value="/home/moxxx066/Workspace/data/bags/KITTI/seq00.bag" />
    <param name="img_topic" value="/left/image_raw" />
    <param name="output_file" value="$(find so_dso_place_recognition)/place_recognition/results/KITTI/00/BoW/dist_mat.txt" />
  </node>
</launch>
